hrjobcontract/js/src/job-contract/controllers/contract-list.js

```diff
@@ -29,7 +29,8 @@ define([
                 },
                 promiseUtils = {
                     hoursLocation: UtilsService.getHoursLocation(),
-                    payScaleGrade: UtilsService.getPayScaleGrade()
+                    payScaleGrade: UtilsService.getPayScaleGrade(),
+                    absenceTypes: UtilsService.getAbsenceTypes()
```

Comment:
Fix the indentation in this bit (since we have to do it incrementally for already existing files anyway)